mteb/models/gme_models.py

```diff
@@ -0,0 +1,488 @@
+from __future__ import annotations
+
+import logging
+import math
+import os
+from functools import partial
+from typing import Any
+
+import torch
+from PIL import Image
+from torch.utils.data import DataLoader
+from tqdm.autonotebook import tqdm
+from transformers import AutoModelForVision2Seq, AutoProcessor
+
+import mteb
+from mteb.encoder_interface import PromptType
+from mteb.model_meta import ModelMeta
+
+from .instructions import DEFAULT_PROMPTS, TASKNAME2INSTRUCTIONS
+
+logging.basicConfig(level=logging.WARNING)
+logger = logging.getLogger(__name__)
+
+HF_GME_QWEN2VL_2B = "Alibaba-NLP/gme-Qwen2-VL-2B-Instruct"
+HF_GME_QWEN2VL_7B = "Alibaba-NLP/gme-Qwen2-VL-7B-Instruct"
+
+
+_ViDoRe_INSTRUCTION = "Find a screenshot that relevant to the user's question."
+_GME_TASK_TO_INSTRUCTION = {
+    "WebQAT2TRetrieval": "Retrieve passages from Wikipedia that provide answers to the following question.",
```

Comment:
FYI, in main brunch prompts are assigned to tasks. In future they should be moved

Reply:
Thanks for the suggestion, But in the MIEB branch, we haven't switched to the new implementation yet.
I will merge gme instructions into  https://github.com/embeddings-benchmark/mteb/blob/10ba68d9683b4412a7bce116f9f9d9c6fb7e6e24/mteb/models/instructions.py#L25